src/index.html

```diff
@@ -22,6 +22,405 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="container">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo header"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+              data-qa="is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li
+            class="link"
+            data-qa="hover"
+          >
+            <a
+              href="index.html"
+              class="nav-item"
+              data-qa="nav-hover"
+            >
+              LAPTOPS &AMP; COMPUTERS
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="index.html"
+              class="nav-item"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="card-container">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
```

Comment:
Ensure that the image path 'images/imac.jpeg' is correct and that the image file exists in the specified directory. This will prevent broken image links on the webpage.